internal/discovery/discovery.go

```diff
@@ -40,7 +41,7 @@ func Run(configs *[]load.Config) {
 			}).Error("discovery: unable to set docker client")
 		} else {
 			ctx := context.Background()
-			containerList, err := cli.ContainerList(ctx, types.ContainerListOptions{})
+			containerList, err := cli.ContainerList(ctx, container.ListOptions{})
```

Comment:
Do we have a test for this?

Reply:
I don't think so, there is a non implemented unit test, and I don't find any other tests, pretty sad TBH. 
Even though that , this is just change implies no more risk that any other version bump that is being done with automerge, since the type was moved from one version to another.

```
// ContainerListOptions holds parameters to list containers with.
//
// Deprecated: use [container.ListOptions].
type ContainerListOptions = container.ListOptions
```